src/Symfony/Component/Console/Application.php

```diff
@@ -250,12 +250,12 @@ public function doRun(InputInterface $input, OutputInterface $output)
         }
 
         try {
-            $this->runningCommand = null;
+            unset($this->runningCommand);
             // the command name MUST be the first element of the input
             $command = $this->find($name);
         } catch (\Throwable $e) {
             if (!($e instanceof CommandNotFoundException && !$e instanceof NamespaceNotFoundException) || 1 !== \count($alternatives = $e->getAlternatives()) || !$input->isInteractive()) {
-                if (null !== $this->dispatcher) {
+                if (isset($this->dispatcher)) {
```

Comment:
why is this better than making the property nullable?
making the property not-nullable might hide the fact that this check is going to be needed before using the propery. Making it nullable would make that explicit, isn't it?